nsxt/utils_test.go

```diff
@@ -459,6 +458,23 @@ resource "nsxt_policy_tier0_gateway" "test" {
 }`, getAccTestResourceName(), edgeClusterName)
 }
 
+func testAccNsxtPolicyTier1WithEdgeClusterForVPN(edgeClusterName string) string {
```

Comment:
do we need `edgeClusterName` parameter here? I don't think its ever anything other than `test`  (same for `testAccNsxtPolicyTier0WithEdgeClusterForVPN`)

Reply:
I agree, but it seems we have several function using the edgeClusterName parameter, and they are used many times, I think we can change this function definition in a seperate PR

Reply:
sounds good